Translation suggestions and New Strings

Knofbath opened the pull request.

Commit:
Translation suggestions and New Strings

wchristian commented on binary_translations.pm:
```diff
@@ -20,20 +20,38 @@ sub data {
-        "配備輸送船数" => { tr => "Cargo ships at sea", },
-        "予想獲得資源" => { tr => "Expected Gain", },
-        "海上護衛艦隊" => { tr => "Sea Escort Fleet", },
+        "配備輸送船数" => { tr => "Transport ships", desc => "Numer of deployed shipping vessels, Cargo ships in area" },
```
can you fix the "Numer" pelase? :D

Commit:
Update binary_translations.pm

Commit:
Merge branch 'master' into patch-1

Commit:
Update binary_translations.pm

miyagami committed:
Update binary_translations.pm

more specific of : 艦隊   状態   艦種   艦船名    Lv    耐久   火力   対空   速カ

miyagami commented:
> Whoops, guess I did mix up HP for Armor.
> 
Happens.

> Could use "Type" instead of "Class", this column will all be abbreviations like DD/CL/CA/CVL/CV.
> 
No, definitely Class. 
A Type refers not to the hull, but to things like Cargo Vessel, Passenger Ship, Warship, etc..
Taken from Wikipedia:
"A ship class is a group of ships of a similar design. This is distinct from a ship type, which might reflect a similarity of tonnage or intended use. "
https://en.wikipedia.org/wiki/Ship_class

> Firepower is the current stat listed on the ships. Though ATK is fine since this is just a sort list.
> 
I agree, Firepower is the current listing. We can change it. However, as i said in chat last night, 火力 literally means FirePower.

> Fleet, ___ , Type , Name , Lv , HP , ATK , AA , SPD.
> 
We can not use HP as the word Health is in use in other places.
**Fleet ??? Class  Name Lv Health Firepower AntiAir Speed**
I shy from these, but : FLT ??? CLS Name Lv Health FP AA SPD they look sloppy.

> I don't know how you should shorten Condition to fit. Status maybe? or CND?
>
I have not found the listing for 状態 [じょうたい] being translated anywhere in game yet, so we should have free reign on how we translate it.
Condition, Current status, State [of being] are all good words for it. But, to keep short.
I honestly do not like doing all the shorthand. I would like us to go for as best we can, so if we can fit State in there, it would be better in my opinion.

Knofbath commented:
HP is a common game abbreviation for Hit Points or Health Points, it certainly won't annoy anyone to shorten it in this scenario.

This is the context we are using it in:
https://i.imgur.com/jiD71BD.png

miyagami commented:
Yes, I am playing the game, and I know what it is being used for.

Consistency is key when localizing things. It has already been established as just "Health".
It would bother me, and therefore I must assume it would bother other people.

I would like to see a build with "Health" so I can see the typesetting. that may just solve the discussion for us.

miyagami committed:
Translation clarifications

Made some extra clarifications and corrections to some translations and context.

Commit:
More strings added

Map screen
Compass Fairy
Battle Formation Select

Knofbath commented:
えいっ and それっ

Maybe we just romanji these to start, since they just sound like shouting?

miyagami committed:
どうぞ。

Some more clarifications and better translations. There are some areas I am saving for future work. Raiding and Expedition sub/sections.

Knofbath commented:
Previous work similar to the expedition screen:
https://i.imgur.com/gJKb3iB.png - (I am aware of the spelling error)

Expedition screen again for reference:
https://i.imgur.com/dxUe6CF.png

Knofbath commented:
Raiding will need to fit into image bubbles like this:
https://cdn.discordapp.com/attachments/235919493686231051/522906582376251403/unknown.png

Something like Escort Sunk! , Escort Damaged! , Transports Sunk ! (the number is overlayed in the middle in red?) , Repair Team Used!

Knofbath commented:
> No, definitely Class.
> A Type refers not to the hull, but to things like Cargo Vessel, Passenger Ship, Warship, etc..
> Taken from Wikipedia:
> "A ship class is a group of ships of a similar design. This is distinct from a ship type, which might reflect a similarity of tonnage or intended use. "
> https://en.wikipedia.org/wiki/Ship_class

Just had a moment of inspiration on this one, definitely "Type" in this case. Because as the wikipedia states "For example, USS Carl Vinson is a nuclear aircraft carrier (ship type) of the Nimitz class (ship class)." The type will be CA, and the ship class isn't shown, such as the Furutaka-class cruisers.

https://kancolle.wikia.com/wiki/Category:Furutaka_Class

miyagami commented:
> Raiding will need to fit into image bubbles like this:
> https://cdn.discordapp.com/attachments/235919493686231051/522906582376251403/unknown.png
> 
> Something like Escort Sunk! , Escort Damaged! , Transports Sunk ! (the number is overlayed in the middle in red?) , Repair Team Used!

I will get those sooner or later.

In the future, when you reference an image from the asset files. please include the pathing to said asset.

miyagami commented:
> > No, definitely Class.
> > A Type refers not to the hull, but to things like Cargo Vessel, Passenger Ship, Warship, etc..
> > Taken from Wikipedia:
> > "A ship class is a group of ships of a similar design. This is distinct from a ship type, which might reflect a similarity of tonnage or intended use. "
> > https://en.wikipedia.org/wiki/Ship_class
> 
> Just had a moment of inspiration on this one, definitely "Type" in this case. Because as the wikipedia states "For example, USS Carl Vinson is a nuclear aircraft carrier (ship type) of the Nimitz class (ship class)." The type will be CA, and the ship class isn't shown, such as the Furutaka-class cruisers.
> 
> https://kancolle.wikia.com/wiki/Category:Furutaka_Class

No, "Class"

https://en.wikipedia.org/wiki/List_of_aircraft_carriers_of_the_United_States_Navy

"Aircraft carriers are warships that act as airbases for carrier-based aircraft. In the United States Navy, these consist of ships commissioned with hull classification symbols CV (aircraft carrier), CVA (attack aircraft carrier), CVB (large aircraft carrier), CVL (light aircraft carrier), CVN (aircraft carrier (nuclear propulsion)) and CVAN (attack aircraft carrier (nuclear propulsion))."

I could pull other material from other sources for the other ship classes as well.

Knofbath commented:
> > > No, definitely Class.
> > > A Type refers not to the hull, but to things like Cargo Vessel, Passenger Ship, Warship, etc..
> > > Taken from Wikipedia:
> > > "A ship class is a group of ships of a similar design. This is distinct from a ship type, which might reflect a similarity of tonnage or intended use. "
> > > https://en.wikipedia.org/wiki/Ship_class
> > 
> > 
> > Just had a moment of inspiration on this one, definitely "Type" in this case. Because as the wikipedia states "For example, USS Carl Vinson is a nuclear aircraft carrier (ship type) of the Nimitz class (ship class)." The type will be CA, and the ship class isn't shown, such as the Furutaka-class cruisers.
> > https://kancolle.wikia.com/wiki/Category:Furutaka_Class
> 
> No, "Class"
> 
> https://en.wikipedia.org/wiki/List_of_aircraft_carriers_of_the_United_States_Navy
> 
> "Aircraft carriers are warships that act as airbases for carrier-based aircraft. In the United States Navy, these consist of ships commissioned with hull classification symbols CV (aircraft carrier), CVA (attack aircraft carrier), CVB (large aircraft carrier), CVL (light aircraft carrier), CVN (aircraft carrier (nuclear propulsion)) and CVAN (attack aircraft carrier (nuclear propulsion))."
> 
> I could pull other material from other sources for the other ship classes as well.

Type is better because this field only contains DD/CL/CA/CAV/BB/BBV/CV/CVL/etc.

Class would be better if the field contains Kamikaza-class or Mutsuki-class, but all that is shown is the generic DD.

List of kancolle classes:
     Destroyer (DD)
        Kamikaze-class
        Mutsuki-class
        Fubuki-class
        Ayanami-class
        Akatsuki-class
        Hatsuharu-class
        Shiratsuyu-class
        Asashio-class
        Kagerou-class
        Yuugumo-class
        Akizuki-class
        Shimakaze
        Type 1934-class
        Maestrale-class
        John C. Butler-class
        J-class
        Tashkent-class
    Coastal Defense Ship (DE)
        Shimushu-class
        Etorofu-class
        Hiburi-class
    Light Cruiser (CL)
        Tenryuu-class
        Kuma-class
        Nagara-class
        Sendai-class
        Yuubari
        Agano-class
        Ooyodo
        Gotland
    Training Cruiser (CT)
        Katori-class
    Torpedo Cruiser (CLT)
        Kuma-class
    Heavy Cruiser (CA)
        Furutaka-class
        Aoba-class
        Myoukou-class
        Takao-class
        Mogami-class
        Tone-class
        Admiral Hipper-class
        Zara-class
    Aviation Cruiser (CAV)
        Mogami-class
        Tone-class
    Battleship (BB)
        Kongou-class
        Fusou-class
        Ise-class
        Nagato-class
        Yamato-class
        Bismarck-class
        Vittorio Veneto-class
        Iowa-class
        Queen Elizabeth-class
        Nelson-class
        Richelieu-class
        Gangut-class
    Aviation Battleship (BBV)
        Fusou-class
        Ise-class
    Light Aircraft Carrier (CVL)
        Light Carriers (Various Types)
        Shouhou-class
        Hiyou-class
        Chitose-class
        Taiyou-class
        Shinyou-class
        Mogami-class
        Casablanca-class
    Regular Aircraft Carrier (CV)
        Aircraft Carriers (Various Types)
        Shoukaku-class
        Unryuu-class
        Graf Zeppelin-class
        Aquila-class
        Lexington-class
        Essex-class
        Ark Royal-class
    Armored Aircraft Carrier (CVB)
        Shoukaku-class
        Taihou-class
        Lexington-class
    Submarine Aircraft Carrier (SSV)
        Type AM
        I-400-class
        Junsen III Type (J3)
        Type-B (B1)
        Type-B Mod.2 (B3)
    Submarine (SS)
        Type Kaidai VIa (KD6)
        Junsen III Type (J3)
        Type-B (B1)
        Type-B Mod.2 (B3)
        I-504
        RO-500
        Type 3 Submergence Transport Vehicle
        UIT-25
        Type IXC
        Marconi-class
    Seaplane Tender (AV)
        Kamoi-class
        Chitose-class
        Mizuho-class
        Akitsushima-class
        Commandant Teste-class
    Submarine Tender (AS)
    Amphibious Assault Ship (LHA)
        Akitsu Maru-class
    Fleet Oiler (AO)
        Kamoi-class
        Revised Kazahaya-class
    Repair Ship (AR)
        Akashi-class

miyagami commented:
> > > > No, definitely Class.
> > > > A Type refers not to the hull, but to things like Cargo Vessel, Passenger Ship, Warship, etc..
> > > > Taken from Wikipedia:
> > > > "A ship class is a group of ships of a similar design. This is distinct from a ship type, which might reflect a similarity of tonnage or intended use. "
> > > > https://en.wikipedia.org/wiki/Ship_class
> > > 
> > > 
> > > Just had a moment of inspiration on this one, definitely "Type" in this case. Because as the wikipedia states "For example, USS Carl Vinson is a nuclear aircraft carrier (ship type) of the Nimitz class (ship class)." The type will be CA, and the ship class isn't shown, such as the Furutaka-class cruisers.
> > > https://kancolle.wikia.com/wiki/Category:Furutaka_Class
> > 
> > 
> > No, "Class"
> > https://en.wikipedia.org/wiki/List_of_aircraft_carriers_of_the_United_States_Navy
> > "Aircraft carriers are warships that act as airbases for carrier-based aircraft. In the United States Navy, these consist of ships commissioned with hull classification symbols CV (aircraft carrier), CVA (attack aircraft carrier), CVB (large aircraft carrier), CVL (light aircraft carrier), CVN (aircraft carrier (nuclear propulsion)) and CVAN (attack aircraft carrier (nuclear propulsion))."
> > I could pull other material from other sources for the other ship classes as well.
> 
> Type is better because this field only contains DD/CL/CA/CAV/BB/BBV/CV/CVL/etc.
> 
> Class would be better if the field contains Kamikaza-class or Mutsuki-class, but all that is shown is the generic DD.
> 
> <SNIP>

And, you are wrong.
Discussion on this particular topic is over.

Knofbath commented:
https://en.wikipedia.org/wiki/Warship#Types_of_warship

wchristian committed:
Merge branch 'master' into patch-1

wchristian commented:
Ok, i tried to use the interface of github to synch this branch up again, but that didn't work quite as expected as github munged the newlines. I think we can close this PR for now in any case, and make new ones for other stuff.

Notable other changes:
 - All fonts are now augmented based on the original fonts. That means some strings got visually wider. We might need to just replace all of them with narrow fonts and hope it doesn't break anything else.
 - I needed to move certain translation entries between the binary and csharp dictionaries. Which one i use depends on where the japanese text is located. I plan to merge the two later on.

Changes i made that you guys will probably want to see are in the following links:
 * https://github.com/wchristian/kc-vita-translation/commit/ccc2d797211f4a59755dd941d8597b020f5cee4c
 * https://github.com/wchristian/kc-vita-translation/commit/60bc2b8b3e6d2238435a3e9f1991376caf8275e1
 * https://github.com/wchristian/kc-vita-translation/commit/8702c1378d087302b1064fde51f7a5b51b6d4440

miyagami commented:
I am seeing some issues right off:
Operation Map:
"Transport s" 
"Escort Flee"

Organization Menu - Thank you, now I can try to come up with a better format based on sizes.
Interesting how Fleet spills over to the left of the table header
Voice chatted with you about some table header justifications. - further discussion pending patch test.
